ignite/handlers/checkpoint.py

```diff
@@ -327,6 +327,18 @@ def __init__(
         self.include_self = include_self
         self.greater_or_equal = greater_or_equal
 
+    def __get_filename_pattern(self, global_step: Optional[int]) -> str:
```

Comment:
@sdesrozis why using dunder `__` instead of single underscore `_` ,  `_get_filename_pattern` ?

Reply:
For privacy but one underscore would be enough.